src/app/config/schema.ts

```diff
@@ -436,6 +437,39 @@ export const prodOnlyVarsSchema: Schema<IProdOnlyVarsSchema> = {
     env: 'DB_HOST',
     sensitive: true,
   },
+  // Rename these env vars to without the _sg suffix when SES migration is over (opengovsg/formsg-private#130)
+  port_sg: {
+    doc: 'SMTP port number',
+    format: 'port',
+    default: null,
+    env: 'SES_PORT_SG',
+  },
+  host_sg: {
+    doc: 'SMTP hostname',
+    format: String,
+    default: null,
+    env: 'SES_HOST_SG',
+  },
+  user_sg: {
+    doc: 'SMTP username',
+    format: String,
+    default: null,
+    env: 'SES_USER_SG',
+  },
+  pass_sg: {
+    doc: 'SMTP password',
+    format: String,
+    default: null,
+    env: 'SES_PASS_SG',
+    sensitive: true,
+  },
+  // Remove this when SES migration is over (opengovsg/formsg-private#130)
+  nodemailer_sg_warmup_start_date: {
+    doc: 'Date where SG nodemailer client will start sending emails',
+    format: String,
+    default: '',
```

Comment:
There should either not be a default, or the default should be some date ridiculously in the future so the default behaviour to be at rollout threshold value of 0 (e.g. `2099-01-01T23:59:59+08:00`)

Reply:
Well... Turns out that in practice an invalid date does have the behaviour of threshold `0`, but I reckon it's by accident rather than a conscious choice, correct? 😅 

As in: if the env var is not representing a valid date, `START_TS` will be an `Invalid Date` with a `getTime()` value of `NaN`. 

The calculations based on that in `getThreshold()` will eventually return `NaN` as well, and an interesting fact of `NaN` is that comparisons to it are **always false** (i.e. `1 < NaN` is false, but `NaN < 1` is also false). Therefore `rand < NaN` will always be false, and in effect that represents a rollout threshold of `0`. 

So again, it works, but it's dirty. It would be better to make the behaviour explicit, and have an explicit handling for `if (isNaN(START_TS)) return 0` as first line of the function `getThreshold()`.